core/src/controllers/rows.js

```diff
@@ -77,8 +77,9 @@ const listTableRows = async (req, res) => {
   // e.g. ?_filters=name:John,age:20
   // will filter by name like '%John%' and age like '%20%'
   let filters = [];
+  const re = /(\w+:?(\[.*?\]|\w+)?)/g;
```

Comment:
could you include a comment immediately before you declare `re` to explain that we need to split the filters by `,` except when in an array, please @TahaKhanAbdalli